src/org/tensorflow/yolo/view/ClassifierActivity.java

```diff
@@ -45,11 +45,21 @@
     private BorderedText borderedText;
     private long lastProcessingTimeMs;
 
+// shimatani
+    private String lastRecognizedClass = "";
```

Comment:
lastRecognizedClass, matchCount, lastResult, and nowResult were made class members because it is necessary to refer to the previously held value each time the run () function is called in the background.

tts can be declared in private functions.

Since masg1 and msg2 need two to display in two lines and I can not think of a way to return two as the return value of the makeTts () function, I made them class members.